src/Kinto.elm

```diff
@@ -219,6 +238,35 @@ encodeData encoder =
 
 
 
+-- Pagination
+
+
+{-| A type for a paginated list of objects. The `nextPage` field may contain the
+URL to request to retrieve the next page of objects, usually using `getNextList`.
+-}
+type alias Pager a =
+    { objects : List a
+    , total : Int
+    , nextPage : Maybe String
+    }
+
+
+decodePager : Decode.Decoder (List a) -> Http.Response String -> Result.Result String (Pager a)
+decodePager decoder response =
+    let
+        nextPage =
+            Dict.get "Next-Page" response.headers
+
+        total =
+            Dict.get "Total-Records" response.headers
+                |> Maybe.map (Result.withDefault 0 << String.toInt)
```

Comment:
I find function composition to be more readable when "reversed" (displayed in the execution order): something like `String.toInt >> Result.withDefault 0`.
It reads in the same order as you would write it using pipes `|>`.

What do you think?